src/components/mobile/survey-questions.tsx

```diff
@@ -122,9 +208,10 @@ export function MobileSurveyQuestionnaire({
     <div>
       <Form {...form}>
         <form
-          onSubmit={form.handleSubmit((data) => {
+          onSubmit={form.handleSubmit(async () => {
```

Comment:
Why have 2 components that do the same state management? The mobile component looks identical.

Reply:
Solved in #46